release/stress_tests/cluster.yaml

```diff
@@ -44,7 +44,7 @@ auth:
 # http://boto3.readthedocs.io/en/latest/reference/services/ec2.html#EC2.ServiceResource.create_instances
 head_node:
     InstanceType: m4.16xlarge
-    ImageId: ami-06d51e91cea0dac8d # Ubuntu 18.04
+    ImageId: ami-042d0d6196f494652 # Custom ami
```

Comment:
can't we just let ray or the provider auto-fill?

Reply:
If we remove this line, is it auto-filled?

Reply:
Yes